### DISCUSSION TITLE
No 4K HDR with Ezcap 269

### DISCUSSION THREAD

**Discussion by Schnielz** [awawa-dev/HyperHDR#510] [Q&A]
Hello All,

I have my HyperHDR setup with my old Ezcap 269 + RPi 4 running since quite a while. Now that I got my PS5, I recognized that it disables HDR when the resoultion ist Auto / 4K. PS5 states that HDR is generally available in the Video settings but 4K SDR image arrives at the TV. Also in games settings HDR configuration is disabled. When I reduce the resolution to 1080p in the PS5 settings, I receive a HDR signal at TV and at my RPi. The strange thing is that the FireTV 4K stick and the PS4 Pro have no problems to deliver 4K/HDR image. I know the Ezcap 269 is not state of the art but I am quite happy with the image and I am hesitating to invest another 60 € for new Splitter and Grapper with no garantee that 4K/HDR will work then as the root cause is pretty unclear. Does anyone have a suggestion what else I could check?

Regards
Nils

**Comment by awawa-dev** [comment]
Hi
Ezcap 269 can only offer HDR for RGB color format if I remember correctly. It's very limited in that matter and will limit the video source as a consequence. Check the video color format because it's critical for this grabber. BTW recommended ezcoo splitter + modern MS2130 grabber is rather a matter of ~45€ to be precise.

**Comment by Schnielz** [comment]
Thanks, that was a good hint. The PS5 uses YUV444 by default. But its has the option to restrict to YUV422 or YUV420 in the video settings (4K video transfer rate -1 or -2). Limiting it to YUV420 solved the issue for me.
But maybe I just invest some €s to get rid of this and the other all the limitations that come with Ezcap 296.
